specs/disk_details_for_hardware_inventory.adoc

```diff
@@ -1,121 +1,319 @@
+            driver_modules: "ahci"
+            driver: ["ahci", "sd"]
+            device_files: "/dev/sda, /dev/disk/by-id/ata-SAMSUNG_MZ7TE512HMHP-000L1_S1GJNSAG400778, /dev/                          disk/by-id/wwn-0x4d30445853885002"
+            device_number: "block 8:0-8:15"
+            device: "/dev/sda"
+            drive_status: "no medium"
+            rmversion: "6L0Q"
+            bios_id: "0x80"
+            geo_bios_edd: "CHS 992277/16/63"
+            geo_bios_legacy: "CHS 1023/255/63"
+            geo_logical: "CHS 62260/255/63"
+            disk_kernel_name: "/dev/sda"
+            major_to_minor_no: "8:0"
+            fstype: "LVM2_member"
+            mount_point: ""
+            label: ""
+            fsuuid: ""
+            read_ahead: "128"
+            read_only: "0"
+            removable_device: "0"
+            size: "10737418240"
+            state: "running"
+            owner: "root"
+            group: "disk"
+            mode: "brw-rw----"
+            alignement: "0"
+            min_io_size: "512"
+            optimal_io_size: "0"
+            phy_sector_size: "512"
+            log_sector_size: "512"
+            ssd: "True"
+            scheduler_name: "cfq"
+            req_queue_size: "128"
+            discard_align_offset: "0"
+            discard_granularity: "0"
+	    discard_max_bytes: "0"
+	    discard_zeros_data: "0"
+	    partition:{
+            		unique_id: "2pkM.SE1wIdpsiiC"
+                        parent_name: "3OOL.iLjwFstLQvC"
+            		disk_type: "partition"
+            		device_name: "/dev/sda1"
+            		sysfs_id: "/class/block/sda/sda1"
+            		device_files: "/dev/sda1, /dev/disk/by-id/ata-SAMSUNG_MZ7TE512HMHP-000L1_S1GJNSAG                                      400778-part1, /dev/disk/by-id/wwn-0x4d30445853885002-part1, /dev/di                                      sk/by-uuid/fff-c5ec-4674-894a-d9ae57b8243c"
+            		disk_kernel_name: "/dev/sda1"
+            		major_to_minor_no: "8:1"
+            		fstype: "ext4"
+            		mount_point: "/boot"
+            		label: ""
+            		fsuuid: "dda9f15f-c5ec-4674-894a-d9ae57b8243c"
+            		read_ahead: "128"
+            		read_only: "0"
+            		removable_device: "0"
+            		size: "10737418240"
+            		state: ""
+            		owner: "root"
+            		group: "disk"
+            		mode: "brw-rw----"
+            		alignement: "0"
+            		min_io_size: "512"
+            		optimal_io_size: "0"
+            		phy_sector_size: "512"
+            		log_sector_size: "512"
+            		ssd: ""
+            		scheduler_name: "cfq"
+            		req_queue_size: "128"
+            		discard_align_offset: "0"
+            		discard_granularity: "0"
+            		discard_max_bytes: "0"
+            		discard_zeros_data: "0"
+		}
+
+	}
+	"disk_id2": {
+		//same as disk_id1 structure
+	}
+   }
+   "all_block_device":{
```

Comment:
@brainfunked it will stored under 
 all_block_device - nodes/$Node_context.node_id/block_devices/all
 used_block_device - nodes/$Node_context.node_id/block_devices/used
  free_block_device  - nodes/$Node_context.node_id/block_devices/free